src/airflow/dags/common_airflow.py

```diff
@@ -164,11 +164,11 @@ def submit_step(cluster_name, step_id):
     return submit_pyspark_job(
         cluster_name=cluster_name,
         task_id=step_id,
-        python_module_path=python_cli,
+        python_module_path=PYTHON_CLI,
```

Comment:
I'd rather pass here the full path to the Python module, rather than building it inside the function. I was struggling yesterday to debug which module exactly we were running, and it was because the logic was buried inside `submit_pyspark_job`
So I suggest something like this:
```suggestion
        python_module_path=f"{INITIALISATION_BASE_PATH}/{PYTHON_CLI}",
```
 or `PYTHON_CLI = f"{INITIALISATION_BASE_PATH}/cli.py`

Reply:
Fair enough! Thank you for the suggestion, I'll accept it now